package exception

class MyException: java.lang.Exception {
 internal constructor(): super()

 internal constructor(cause: kotlin.Throwable?): super(cause)
}
